packages/beacon-node/test/e2e/db/api/beacon/repositories/blockArchive.test.ts

```diff
@@ -70,49 +69,4 @@ describe("BlockArchiveRepository", function () {
     savedBlock2.message.slot = sampleBlock.message.slot;
     expect(ssz.phase0.SignedBeaconBlock.equals(savedBlock1, savedBlock2)).to.equal(true);
   });
-
-  it("batchPutBinary should be faster than batchPut", async () => {
```

Comment:
This is not a test but a very old performance test with winston profiler. Removed as it's not even tracked